2.1.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <meta charset="UTF-8" ; />
+        <body>
+                <p>The highest number is <span id="demo"></span>.</p>
+            <script>
+                var num = [110, 220, 3030, 4040, 500500];
+                document.getElementById("demo").innerHTML = myArrayMax(num);
+            function myArrayMax(arr){
+                let len = arr.length
+                let max= - Infinity;
```

Comment:
First table element can be max instead of - infinity, but it is also good solution so it don't need to be changed.